src/http.c

```diff
@@ -479,10 +509,52 @@ http_stream_setup(char **stream, const char *url)
   /* Read the playlist until the first stream link is found, but give up if
    * nothing is found in the first 10 lines
    */
+  in_playlist = false;
   n = 0;
   while ((line = evbuffer_readln(ctx.input_body, NULL, EVBUFFER_EOL_ANY)) && (n < 10))
     {
+      // Skip comments and blank lines without counting for the limit
+      if (pl_format == PLAYLIST_M3U && (line[0] == '#' || line[0] == '\0'))
```

Comment:
Seems good to skip these lines, but why only for m3u?

Reply:
Because I could not find the comment syntax for PLS.
It looks like an INI file, so it might be a semicolon not a hash mark (https://stackoverflow.com/questions/1378219/do-standard-windows-ini-files-allow-comments).